### DISCUSSION TITLE
스프린트 1 진행 방법

### DISCUSSION THREAD

**Discussion by gracefulBrown** [woowacourse/prolog#850] [Notice]
# 이슈 진행 방법
1. 매칭된 페어와 할당 받은 이슈를 확인한다.
2. 코치와 멘토에게 선택한 이슈에 대해서 소개한다. (슬랙을 통해 소통할 수 있도록 한다)
3. 멘토에게 이슈 진행 방법에 대해 질문하고 가이드를 받는다.
4. 코치들에게 코드 설명이나 개발 방식에 대해 질문하고 설명을 듣는다.
5. 이슈를 진행한다. 진행간 질문은 멘토에게 하고 구체적인 방법 및 기술 설명은 코치들에게 듣는다.
6. 코드 작업이 끝나면 PR을 올리고 리뷰를 기다린다.
7. 멘토에게 코드리뷰를 받고 반영하고 머지한다.
8. 끗 :)

# 페어, 멘토, 코치 매칭 결과
- 공개 예정

# 사전 학습이 필요한 내용
### 프론트엔드
- 리액트

### 백엔드
> 이슈 선택 후 날 잡고 소개하는 시간을 가지겠습니다. (강의는 아니고 학습법 안내)
- Spring Test(인수 테스트, 문서화 테스트 등)
- Spring Data JPA

**Comment by pocojang** [comment]
## 프론트엔드

### 멘토와 멘티 크루 오프라인 미팅

- 매칭 및 결과 공유
- 사전 학습 내용 확인
- 이슈 진행 방법 안내
